runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerPlugin.java

```diff
@@ -360,4 +374,16 @@ protected void markTile(LocalPoint localPoint)
 
 		loadPoints();
 	}
+
+	private void clearPoints()
+	{
+		points.clear();
```

Comment:
Hmm, maybe this should clear only current region, so the behaviour would be at least deterministic.

Reply:
This is only clearing for the current region. When you change regions points is loaded with that regions marked tiles. I actually couldn't find a smart way to clear all regions at once so I left it at just this.

Reply:
But this looks like it is clearing current region and all other regions that are visible to you (eg the surrounding ones I guess)

Reply:
Yes, but why would you clear just the region you are in if you don't know where the region lines are? This is just clearing all visible markers..

Reply:
Hmm, true